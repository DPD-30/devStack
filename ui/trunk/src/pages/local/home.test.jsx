import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesLocalHome from 'pages/local/home.jsx'

jest.unmock('pages/local/home.jsx')


describe('', () => {
    it('PagesLocalHome should match snapshot', () => {
        const comp = renderToString(<PagesLocalHome />)

        expect(comp).toMatchSnapshot()
    })
})
