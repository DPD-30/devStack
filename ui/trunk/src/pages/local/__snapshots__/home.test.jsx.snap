// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesLocalHome should match snapshot 1`] = `
<div data-reactroot>
  Local user account home
</div>
`;
